conda_build/source.py

```diff
@@ -825,16 +884,16 @@ def try_apply_patch(patch, patch_args, cwd, stdout, stderr):
 
             try:
                 try_apply_patch(patch_exe, patch_args,
-                                cwd=src_dir, stdout=stdout, stderr=stderr)
+                                cwd=sub_folder, stdout=stdout, stderr=stderr)
             except Exception as e:
                 exception = e
         if exception:
             raise exception
     return attributes_output
 
 
-def apply_patch(src_dir, patch, config, git=None):
-    apply_one_patch(src_dir, os.path.dirname(patch), os.path.basename(patch), config, git)
+def apply_patch(src_dir, patch, config, git_exe=None):
+    apply_one_patch(src_dir, os.path.dirname(patch), os.path.basename(patch), config, git_exe)
```

Comment:
PyPI skeleton tests failed because the `apply_one_patch` call doesn't include the added `submodules` parameter here (hence `submodules=config` in this call).
It'd probably be good to pass config as a kwarg here to avoid the unintentional alias ;).